.github/workflows/coverage.yml

```diff
@@ -20,44 +20,20 @@ jobs:
     runs-on: ubuntu-latest
     env:
       CARGO_TERM_COLOR: always
-      # PG_* variables are used by psql
```

Comment:
why remove the service here? How is this related to the test coverage?

Reply:
The just based coverage tests already do this.
No need to do this in the GitHub yaml format too.

Can re-add if you prefer